components/gitpod-protocol/data/gitpod-schema.json

```diff
@@ -159,11 +159,6 @@
             "type": "string",
             "description": "Path to where the IDE's workspace should be opened."
         },
-        "privileged": {
```

Comment:
it should be also removed from public scheme: components/dashboard/public/schemas/gitpod-schema.json